airbyte-cdk/bulk/toolkits/extract-cdc/build.gradle

```diff
@@ -8,17 +8,19 @@ dependencies {
     implementation 'io.debezium:debezium-storage-file'
     implementation 'commons-io:commons-io:2.17.0'
 
-    testImplementation testFixtures(project(':airbyte-cdk:bulk:core:bulk-cdk-core-base'))
-    testImplementation testFixtures(project(':airbyte-cdk:bulk:core:bulk-cdk-core-extract'))
-    testImplementation platform('org.testcontainers:testcontainers-bom:1.20.2')
-    testImplementation 'com.alibaba:dns-cache-manipulator:1.8.3' // required for testing with mongo
+    testFixturesImplementation testFixtures(project(':airbyte-cdk:bulk:core:bulk-cdk-core-base'))
+    testFixturesImplementation testFixtures(project(':airbyte-cdk:bulk:core:bulk-cdk-core-extract'))
+    testFixturesImplementation platform('org.testcontainers:testcontainers-bom:1.20.2')
+    testFixturesImplementation 'com.alibaba:dns-cache-manipulator:1.8.3' // required for testing with mongo
     testImplementation 'io.debezium:debezium-connector-mysql'
     testImplementation 'io.debezium:debezium-connector-postgres'
     testImplementation 'io.debezium:debezium-connector-mongodb'
-    testImplementation 'io.debezium:debezium-testing-testcontainers' // required for testing with mongo
-    testImplementation 'io.mockk:mockk:1.13.12'
+    testImplementation 'io.debezium:debezium-connector-sqlserver'
+    testFixturesImplementation 'io.debezium:debezium-testing-testcontainers' // required for testing with mongo
+    testFixturesImplementation 'io.mockk:mockk:1.13.12'
     testImplementation 'org.testcontainers:mysql'
     testImplementation 'org.testcontainers:postgresql'
+    testImplementation 'org.testcontainers:mssqlserver:1.19.0'
```

Comment:
can you add a comment explaining why you're not using the version of `org.testcontainers:mssqlserver` from the bom, please?